.sequelizerc

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+module.exports =  {
+  'config': path.resolve('src','config', 'config.js'),
+  'models-path': path.resolve('src','models'),
+  'seeders-path': path.resolve('src','seeders'),
+  'migrations-path': path.resolve('src','migrations')
+}
```

Comment:
Add a new line for consistency with other files. You can configure your editor to automatically do this.